ruby_event_store/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 Further changes can be tracked at [releases page](https://github.com/RailsEventStore/rails_event_store/releases).
 
+### 0.15.0 (not released yet)
```

Comment:
We've moved changelog to https://github.com/RailsEventStore/rails_event_store/releases as the 1st line says :)